frontend/src/common/enums/app/app-route.enum.ts

```diff
@@ -10,7 +10,7 @@ enum AppRoute {
   SC = '/sc',
   SC_CONFIGURATE_INSTANCE = '/sc/configurate-instance',
   SLC = '/slc',
-  SLC_CONFIGURATE_FUNCTION = '/bs/configurate-function',
+  SLC_CONFIGURATE_FUNCTION = '/slc/configurate-function',
```

Comment:
```suggestion
  SLC_CONFIGURATE_FUNCTION = '/slc/configurate-function/:id?',
```

Reply:
Optional parameters don`t supported in react route v6
https://github.com/remix-run/react-router/issues/7285